crates/matrix-sdk-sqlite/src/state_store.rs

```diff
@@ -1133,8 +1238,74 @@ impl StateStore for SqliteStateStore {
                             })
                     };
 
-                    let encoded_room_id = this.encode_key(keys::STATE_EVENT, &room_id);
-                    let mut room_version = None;
+                    // Clean up raw events we saved for roominfo
+                    if room_infos.get(room_id).is_some() {
```

Comment:
Why do we need this at all? :fearful: Do we need something similar for edits? If we actually have to keep this, can we commonize the code with the indexeddb implementation?